sys/random/mersenne.c

```diff
@@ -129,7 +133,12 @@ uint32_t genrand_uint32(void)
     y ^= (y << 7) & 0x9d2c5680UL;
     y ^= (y << 15) & 0xefc60000UL;
     y ^= y >> 18;
+
+#ifdef MODULE_ENTROPY
+    return y + entropy;
```

Comment:
> (typing, perturbed wireless signals, other hardware fluctuations)

Just keep in mind, this (typing, perturbed wireless signals, other hardware fluctuations) is what e.g., Linux is using as entropy. It is very probably better than nothing, and when it will be proven useless one day, we won't be able to use it anymore. But until then, it (probably) improves our "randomness".